content/blog/eng/v-protetto.md

```diff
@@ -0,0 +1,62 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-18"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+<div id="user-content-toc">
+  <ul>
+    <summary><h1 style="display: inline-block;">Venerdì Protetto presentation</h1></summary>
+  </ul>
+</div>
+
+## What is it all about?
+
+At Facile.it, we strongly believe in sharing knowledge and experiences. Thus, we dedicate one Friday a month to expanding our knowledge and learning among colleagues in the IT department. During the so-called Venerdì Protetto, i. e. Protected Friday, we welcome speakers and participate in talks on IT-related topics. Speakers are colleagues or external guests.
+
+    
+    
+## Curious to learn more?
+
+Click the links to find the talks of each Venerdì Protetto.
+
+<table>
+    <thead>
+        <tr
+            align="left"
+            >
+            <th>Date</th>
+            <th>Topic</th>
+            <th>Speakers</th>
+            <th>Company</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=4><a href="https://engineering.facile.it/blog/eng/venerdi-protetto-14-04-2023/" title="Venerdì Protetto 14-04-2023">14/4/2023</a></td>
+        </tr>
+        <tr>
+            <td><strong>Backstage</strong></td>
+            <td>Lee Mills</td>
+            <td>Spotify</td>
+        </tr>
+        <tr>
+            <td><strong>Code revolution</strong></td>
+            <td>Mirko Urru & Stefano Giurgiano</td>
+            <td>Facile.it</td>
+        </tr>
+        <tr>
+            <td rowspan=2><strong>Dynamic forms</strong></td>
+            <td>Giovanni Fiorentino</td>
+            <td>Facile.it</td>
+        </tr>
+    </tbody>
+</table>

```

Comment:
Wouldn't a table like this require a lot of rework each time that we add a new Venerdì Protetto post? Maybe we could just link to the "Venerdì protetto" category page?

Reply:
Thank you for the suggestion. It really was silly of me. I removed the table and added a link to the VP category.

Would a simple table containing only the talks of the last VP be a good solution, or would it be too difficult to maintain?